docs/itemset_dbc.md

```diff
@@ -0,0 +1,795 @@
+
+### ItemID_7
+
+[Entry](item_template#entry) of the item for the item set [ItemID_7](#itemid_7).
+
+### ItemID_8
+
+[Entry](item_template#entry) of the item for the item set [ItemID_8](#itemid_8).
+
+### ItemID_9
+
+[Entry](item_template#entry) of the item for the item set [ItemID_9](#itemid_9).
+
+### ItemID_10
+
+[Entry](item_template#entry) of the item for the item set [ItemID_10](#itemid_10).
+
+### ItemID_11
+
+[Entry](item_template#entry) of the item for the item set [ItemID_11](#itemid_11).
+
+### ItemID_12
+
+[Entry](item_template#entry) of the item for the item set [ItemID_12](#itemid_12).
+
+### ItemID_13
+
+[Entry](item_template#entry) of the item for the item set [ItemID_13](#itemid_13).
+
+### ItemID_14
+
+[Entry](item_template#entry) of the item for the item set [ItemID_14](#itemid_14).
+
+### ItemID_15
+
+[Entry](item_template#entry) of the item for the item set [ItemID_15](#itemid_15).
+
+### ItemID_16
+
+[Entry](item_template#entry) of the item for the item set [ItemID_16](#itemid_16).
+
+### ItemID_17
+
+[Entry](item_template#entry) of the item for the item set [ItemID_17](#itemid_17).
+
+### SetSpellID_1
+
+[Entry](spell#entry) of the Spell that's used on [ItemID_1](#itemid_1).
+
+### SetSpellID_2
+
+[Entry](spell#entry) of the Spell that's used on [ItemID_2](#itemid_2).
+
+### SetSpellID_3
+
+[Entry](spell#entry) of the Spell that's used on [ItemID_3](#itemid_3).
+
+### SetSpellID_4
+
+[Entry](spell#entry) of the Spell that's used on [ItemID_4](#itemid_4).
+
+### SetSpellID_5
+
+[Entry](spell#entry) of the Spell that's used on [ItemID_5](#itemid_5).
+
+### SetSpellID_6
+
+[Entry](spell#entry) of the Spell that's used on [ItemID_6](#itemid_6).
+
+### SetSpellID_7
+
+[Entry](spell#entry) of the Spell that's used on [ItemID_7](#itemid_7).
+
+### SetSpellID_8
+
+[Entry](spell#entry) of the Spell that's used on [ItemID_8](#itemid_8).
+
+### SetThreshold_1
+
+How many pieces of the Item Set you need refering to [SetSpellID_1](#setspellid_1)
+
+### SetThreshold_2
+
+How many pieces of the Item Set you need refering to [SetSpellID_2](#setspellid_2)
+
+### SetThreshold_3
+
+How many pieces of the Item Set you need refering to [SetSpellID_3](#setspellid_3)
+
+### SetThreshold_4
+
+How many pieces of the Item Set you need refering to [SetSpellID_4](#setspellid_4)
+
+### SetThreshold_5
+
+How many pieces of the Item Set you need refering to [SetSpellID_5](#setspellid_5)
+
+### SetThreshold_6
+
+How many pieces of the Item Set you need refering to [SetSpellID_6](#setspellid_6)
+
+### SetThreshold_7
+
+How many pieces of the Item Set you need refering to [SetSpellID_7](#setspellid_7)
+
+### SetThreshold_8
+
+How many pieces of the Item Set you need refering to [SetSpellID_8](#setspellid_8)
+
+### RequiredSkill
+
+[ID](skillline#id) of the Skill that's required of the Item Set.
+
+### RequiredSkillRank
+
+The required skill rank the player needs to have to use this Item Set.
+
+### DBC File Content
```

Comment:
delete this. DBC is copyrighted by Blizzard. And we do not need to share the contents of it anyways as you can simply open a dbc editor.

Reply:
Done